edx/api-manager
# feat: TNL-10746 CMS API on omnibus plan

Per recent TNL agreement, we'll be hosting our CMS API on the existing ominbus plan in the API Gateway

These changes add the CMS API to the omnibus swagger YAML description

Also, what was formerly a "studio_host" command line argument now becomes "cms_host". This will be a breaking change on the GoCD script activated on merge and will need fixing there.

To test:
Currently, the GoCD pipeline creates a new API Gateway deployment with the following routes in the "Open edX" API configuration object: /catalog, /enterprise, /heartbeat, ./oauth2, and /registrar. Post-this merge, GoCD needs to continue running without error, but the set of routes on the "Open edX" API objects (both for stage and prod) should include a new /cms/v1 path
